infoblox_client/objects.py

```diff
@@ -844,6 +844,7 @@ class ARecord(ARecordBase):
     _fields = ['ipv4addr', 'name', 'view', 'extattrs']
     _search_for_update_fields = ['ipv4addr', 'view']
     _all_searchable_fields = _search_for_update_fields + ['name']
+    _return_fields = _fields
```

Comment:
Please explicitly specify the list of fields here, instead of referencing `_fields`. `_return_fields` is typically a subset of `_fields` - even though they maybe the same for the time being.

Also, please make the same change to `AAAARecord`.